frontend/src/components/Simulator/TransactionsList.vue

```diff
@@ -38,7 +43,10 @@ const handleClearTransactions = () => {
 
 <template>
   <PageSection data-testid="latest-transactions">
-    <template #title>Transactions</template>
+    <template #title
+      >Transactions ({{ accountsStore.transactionCount }})</template
```

Comment:
!!! This is not rendering properly. Please fix before merging

Reply:
this was for testing purpose and forgot to remove it after I deleted the variable - great catch, thanks!